docs/specs/escape.yml

```diff
@@ -162,11 +162,11 @@ outputs:
           <p>
             Link to <a href='a.json'>text</a>
             Link to <a href='a.json'>&lt;some text &amp;&gt;\n</a>
```

Comment:
These two seem should have same localization behavior:
```
    Link to <xref href="a&b?displayProperty=description" />
    Link to <xref:a&b?displayProperty=description>
```

Do you plan to fix this in a later dev task?